components/Table/TableRowColumn.jsx

```diff
@@ -1,10 +0,0 @@
-import React from 'react';
-import { TableRowColumn as MaterialTableRowColumn } from 'material-ui/Table';
-import { getWidthStyle } from './tableHelpers';
-
-export default ({ row, column, totalWidth }) => (
-  <MaterialTableRowColumn style={getWidthStyle(column.width, totalWidth)}>
-    {row && column.displayFn && column.displayFn({ row, column, field: row[column.field] })}
-    {row && row[column.field] && !column.displayFn && (row[column.field].display)}
-  </MaterialTableRowColumn>
-);
```

Comment:
any reason we are doing this? Less maintainable in my opinion to create a huge table component, and the table is concerned with layout. ONly the table row column needs to care about data.